dayrepo/snippets/static/js/process_add.js

```diff
@@ -1,9 +1,28 @@
-let i = 1 ;
+const limit = 6;
+const bool_select = 8;
+const not_select = 10;
 const addForm = () => {
-  let content_area = document.getElementById("form_area");
+  const count = Number(document.getElementById("count").innerHTML);
+  let now_count = count;
+  if (now_count == limit){
+    alert("これ以上増やせません")
+    return
+  }
+  alert("工程を追加します");
+  now_count += 1;
+  const content_area = document.getElementById("form_area");
   let clone_element = content_area.cloneNode(true);
-  clone_element.id = i;
-  alert("工程を追加しますか？");
-  content_area.after(clone_element);
-  i++ ;
+  let content_clone = document.getElementById("add_space");
+  const content_count = content_area.childElementCount;
+  for (let i=0; i<content_count; i++){
+    if (i != bool_select && i <= not_select){
+      clone_element.children[i].children[1].value="";
+    }
+    else if (i == bool_select){
+      clone_element.children[i].children[1].checked="";
+    }
+  }
+  content_clone.before(clone_element)
```

Comment:
細かいですが、基本的に変数は出来るだけ使用する箇所の近くに置きたいです〜
認知負荷も下がります

```suggestion
  const content_count = content_area.childElementCount;
  for (let i=0; i<content_count; i++){
    if (i != bool_select && i <= not_select){
      clone_element.children[i].children[1].value="";
    }
    else if (i == bool_select){
      clone_element.children[i].children[1].checked="";
    }
  }
  
  let content_clone = document.getElementById("add_space");
  content_clone.before(clone_element)
```

Reply:
対応しました
https://github.com/bob-g12/dayrepo-for-transportation/pull/54/commits/951e5383fc8247294b02a6cb3fcbbe7434089940

Reply:
そのコミットの中で、content_clone が対応されていないです〜

それで言うと、content_clone と

Reply:
対応しました
https://github.com/bob-g12/dayrepo-for-transportation/pull/54/commits/76c3d6032c58d351610d5ceeffbdc6d8ba925bf4